src/main/kotlin/org/rust/ide/annotator/fixes/ExpandStructFieldsFix.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.annotator.fixes
+
+import com.intellij.codeInspection.LocalQuickFixAndIntentionActionOnPsiElement
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import org.rust.ide.utils.expandStructFields
+import org.rust.ide.utils.fillRsPatWithFields
+import org.rust.ide.utils.getTupleStructMissingFields
+import org.rust.lang.core.psi.*
+import org.rust.lang.core.psi.ext.*
+import org.rust.openapiext.buildAndRunTemplate
+import org.rust.openapiext.createSmartPointer
+import org.rust.stdext.mapToSet
+
+class ExpandStructFieldsFix(
+    element: PsiElement,
+    private val declarationFieldNames: List<String>
+) : LocalQuickFixAndIntentionActionOnPsiElement(element) {
+    override fun getText() = "Expand actual struct fields"
+
+    override fun getFamilyName() = text
+
+    override fun invoke(project: Project, file: PsiFile, editor: Editor?, pat: PsiElement, endElement: PsiElement) {
+        val factory = RsPsiFactory(project)
+        if (pat is RsPatStruct) {
+            val hasTrailingComma = pat.rbrace.getPrevNonCommentSibling()?.elementType == RsElementTypes.COMMA || pat.dotdot != null
```

Comment:
```suggestion
            val hasTrailingComma = pat.rbrace.getPrevNonCommentSibling()?.elementType == RsElementTypes.COMMA
```

Looks like ` || pat.dotdot != null` is redundant